### PR TITLE
adds  cubew 4.8.1, cubelib 4.8.1 and cubegui 4.8.1, 4.8.2

### PR DESCRIPTION
**Pull request by swat-jsc**

### CODE REVIEW

File: var/spack/repos/builtin/packages/cube/package.py [hunk 1]
```diff
@@ -16,7 +16,10 @@ class Cube(AutotoolsPackage):
 
     homepage = "https://www.scalasca.org/software/cube-4.x/download.html"
     url = "https://apps.fz-juelich.de/scalasca/releases/cube/4.4/dist/cubegui-4.4.2.tar.gz"
+    maintainers = "swat-jsc"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/cube/package.py]: This syntax is no longer supported.  You need to use the new directive described at docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers.

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Confirmed all new version sha256 but the maintainers directive needs to be used.

**Review by michaelkuhn — APPROVED** [review]
LGTM

**Review by tldahlgren — APPROVED** [review]
Reconfirmed all of the new version sha256, rest LGTM.

**Pull request merged**